quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -245,7 +251,7 @@ impl RecoveryEpoch {
 
     fn detect_and_remove_lost_packets(
         &mut self, loss_delay: Duration, pkt_thresh: u64, now: Instant,
-        newly_lost: &mut Vec<Lost>,
+        newly_lost: &mut Vec<Lost>, epoch: packet::Epoch,
```

Comment:
Note the new `epoch` field is only necessary for the qlogging, and so generates a CI error when the `qlog` feature is not enabled. There's ways to workaround that but it suggests we shouldn't be using `epoch` to determine packet type.

Reply:
Any way to annotate to avoid the error?  _epoch?